src/LoginBrowser.php

```diff
@@ -19,18 +18,24 @@ public static function getUserAgent()
         return filter_input(INPUT_SERVER, 'HTTP_USER_AGENT') ?: null;
     }
     
-    public static function supportsU2f($userAgent)
+    // TODO: Replace this with client-side feature detection.
```

Comment:
Would it work to default to webauthn if the user has it, then detect locally, and if not supported redirect to the next option?

Reply:
Hmm, maybe so. That seems like a good user experience.

Reply:
Logged an issue for us to come back to this: https://github.com/silinternational/simplesamlphp-module-mfa/issues/69